react-native-libraries.json

```diff
@@ -9580,5 +9580,11 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/gbumps/react-native-screenguard",
+    "npmPkg": "react-native-screenguard",
```

Comment:
Hello @gbumps, thank you for adding new entry to the directory! 👍 

```suggestion
```

In case of your library we can skip the npm name, since it matches the GitHub repository, otherwise LGTM!